lib/plugins/aws/deploy/lib/extendedValidate.js

```diff
@@ -61,8 +61,7 @@ module.exports = {
       this.serverless.service.getAllFunctions().forEach(functionName => {
         const functionObject = this.serverless.service.getFunction(functionName);
         const individually =
-          (_.has(functionObject, ['package', 'individually']) &&
-            functionObject.package.individually) ||
+          (functionObject.package && functionObject.package.individually) ||
```

Comment:
Let's replace it with `_.get(functionObject, ['package', 'individually'])`

(Note that in future we will replace `_.get` usage with native optional chaining)